CHANGELOG.md

```diff
@@ -3,6 +3,117 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.11
+
+_Dec 8, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Add dragging support for `DateRangePicker` (#6763) @LukasTy
+- ✨ Add missing lazy-loading feature to `DataGridPremium` (#7124) @m4theushw
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v6.0.0-alpha.11` / `@mui/x-data-grid-pro@v6.0.0-alpha.11` / `@mui/x-data-grid-premium@v6.0.0-alpha.11`
+
+#### Breaking changes
+
+- `filterPanelOperators` translation key was renamed to `filterPanelOperator` (#7062) @MBilalShafi
+
+#### Changes
+
+- [DataGrid] Add `GridCell` change in migration guide (#7087) @MBilalShafi
```

Comment:
```suggestion
- [docs] Add `GridCell` change in migration guide (#7087) @MBilalShafi
```

And move to the doc, unless it has changes on the package itself (in which case we need to update the description of the PR)

Reply:
Yes, it's docs only change, sorry about the wrong label 🙈